core/src/ldml/ldml_markers.cpp

```diff
@@ -19,6 +19,10 @@ namespace core {
 namespace ldml {
 
 
+const std::u32string REGEX_PREFIX       = U"\\uffff\\u0008";  // does not include '\\' because it might be a range
```

Comment:
I don't really understand the comment `// does not include '\\' because it might be a range`

Reply:
I'll update the comment.
The whole sequence is (unescaped): `\uffff\u0008\uNNNN` where NNNN is a marker number.  The comment is trying to say that the `\u` part of `\uNNNN` is static, but it's not part of the match because it could also be `\uffff\u0008[...` if it's a "match any marker" `\m{.}` sequence.

Reply:
Gotcha I understand now.